### PR TITLE
Add stats transfering on cloning

### LINKED ISSUE #2746: Stats are lost on cloning
Newly cloned bodies start with nothing in stats

these should be stored on the mind datum probably

### PR DESCRIPTION
**Pull request by hitsedesen**
closes #2746

### CODE REVIEW

File: code/modules/core_implant/cruciform/modules.dm [hunk 1]
```diff
@@ -69,6 +70,7 @@
 	if(ishuman(implant.wearer))
 		var/mob/living/carbon/human/H = implant.wearer
 		dna = H.dna
+		stats = H.stats
```
Comment (LethalGhost) [inline code/modules/core_implant/cruciform/modules.dm]: That's possible what smth will (or already) affect dead body stats.
It's better to make copy of it (values only, not mods).

**Comment by NanakoAC** [comment]
Cruciform is the wrong place for this. Store it in the mind datum instead

the cruciform stores a reference to that

**Comment by NanakoAC** [comment]
and a direct reference is better than making a clone of the stats, because it will be kept updated. We'll be adding ways for players to change their stats during a round

**Comment by LethalGhost** [comment]
> a direct reference is better than making a clone of the stats

@nanako
But that will bound dead body stats to cloned person one.

**Comment by NanakoAC** [comment]
oh no, there has been a terrible misunderstanding here.

I didn't mean move all stat interactions to the mind, just have mind hold a pointer to it
Stats still need to be on the mob for the sake of mindless NPCs which still need stats support

And Ghost, i misunderstood what you meant. A clone should be made, but only at the time the new body is created, so that was a misunderstanding too. i think this is the way things were before

**Comment by NanakoAC** [comment]
I remain convinced that the mind datum is the best place for this, and that this PR is ready to go

**Pull request merged**